glossary.yml

```diff
@@ -5441,23 +5531,24 @@
   en:
     term: "UTF-8"
     def: >
-      A way to store the numeric codes representing Unicode characters in memory that
+      A way to store the numeric codes representing [Unicode](#unicode) characters in memory that
       is [backward-compatible](#backward_compatible) with the older [ASCII](#ascii) standard.
 
 
 - slug: variable_arguments
   ref:
-    - keyword_arguments
+    - keyword_argument
     - named_argument
   en:
     term: "variable arguments"
     def: >
-      In a function, the ability to take any number of arguments. R uses `...` to
-      capture the "extra" arguments.
+      In a function, the ability to take any number of arguments. [R](#r_language) uses `...` to
+      capture the "extra" arguments. [Python](#python) uses `*args` and `**kwargs` to capture
```

Comment:
Shouldn't `**kwargs` be part of keyword arguments instead of named arguments?

Reply:
Kind of a difficult question. In Python `named arguments` are not a thing, but `keyword arguments` are, and they must be named. I am unsure if this is simply a difference in terminology, or if there are some languages that have both `named arguments` and `keyword arguments`.

I put it here, because this is the entry for `variable (numbers of) arguments`, and `**kwargs` is how a variable number of `keyword arguments` is listed in a Python function definition.

If `**kwargs` needs to be mentioned in `keyword arguments` then, by the same logic, `**args` needs to be mentioned in the entry for `argument`.

Reply:
Agreed that it is a difficult question. I'm fine to leave this as-is until someone can come up with a clean solution.